src/util.rs

```diff
@@ -144,3 +162,31 @@ impl<'a> Deref for SchemaAsIpc<'a> {
         &self.pair
     }
 }
+
+// Convert `RecordBatch`es to wire protocol `FlightData`s
+pub fn batches_to_flight_data(
+    schema: &Schema,
+    batches: Vec<RecordBatch>,
+) -> Result<Vec<FlightData>, ArrowError>
+{
+    let options = IpcWriteOptions::default();
+    let schema_flight_data: FlightData = crate::util::SchemaAsIpc::new(schema, &options).into();
```

Comment:
nit: Why not just add SchemaAsIpc to the existing `use crate::util`?

Reply:
Done. Removed `crate::util`